tests/mock/vscode.ts

```diff
@@ -1077,6 +1100,88 @@ export class VSCode {
       d.dispose();
   }
 
+  async webViewsByPanelType(viewType: string) {
+    return (await Promise.all(this._webViewsByPanelType.get(viewType) ?? [])).filter(p => !p.isClosed());
+  }
+
+  private _createWebview(): Webview & { pagePromise?: Promise<Page>, show: () => Promise<Page>} {
```

Comment:
Let's extract / review / land the testing harness separately, before your change.

Reply:
Should I create a new PR for it?

Reply:
I created a new PR: https://github.com/microsoft/playwright-vscode/pull/503